Pod/Classes/NJHNavigationBarSelectorPageViewController.m

```diff
@@ -58,6 +59,7 @@ - (void)viewWillAppear:(BOOL)animated {
         for (UIScrollView *view in self.view.subviews) {
             if ([view isKindOfClass:[UIScrollView class]]) {
                 self.pageScrollView = view;
+                self.pageScrollView.backgroundColor = self.scrollViewBackgroundColor;
```

Comment:
This seems like a weird way of setting the background color of the container scrollview within the pagecontroller. I thought it was "clearColor" before and thus maybe a more appropriate way of setting the color would be to set the this?

![skjermbilde 2015-08-08 kl 11 05 03](https://cloud.githubusercontent.com/assets/1769314/9149833/5b343982-3dbd-11e5-9c88-466c02c9e703.png)

I might be misunderstanding what you are doing here :)